Resources/Prototypes/_Goobstation/Changeling/Entities/Objects/Weapons/Throwable/throwing_stars.yml

```
@@ -10,3 +10,7 @@
       map: ["base"]
   - type: TimedDespawn
     lifetime: 30
+  - type: HolidayVisuals
+    holidays:
+      base:
+      - FestiveSeason
```

yamllint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)